identity/issuer/issuer.go

```diff
@@ -22,12 +25,20 @@ import (
 	"github.com/iden3/go-iden3-crypto/poseidon"
 	"github.com/iden3/go-iden3-crypto/utils"
 
+	"github.com/iden3/go-circom-prover-verifier/parsers"
+	"github.com/iden3/go-circom-prover-verifier/prover"
+	zktypes "github.com/iden3/go-circom-prover-verifier/types"
+	witnesscalc "github.com/iden3/go-circom-witnesscalc"
+	cryptoUtils "github.com/iden3/go-iden3-crypto/utils"
+	wasm3 "github.com/iden3/go-wasm3"
+
 	log "github.com/sirupsen/logrus"
 )
 
 var (
 	ErrIdenGenesisOnly           = fmt.Errorf("Identity is genesis only")
```

Comment:
From the previous commit, I understand that the idea is to use lowercase for errors

Reply:
Yes! updated.